libc/socket/inet-socket.c

```diff
@@ -0,0 +1,172 @@
+/*
+ * Phoenix-RTOS
+ *
+ * test-libc-socket
+ *
+ * inet socket tests
+ *
+ * Copyright 2021, 2024 Phoenix Systems
+ * Author: Ziemowit Leszczynski, Adam Debek
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <errno.h>
+#include <sys/uio.h>
+#include <sys/wait.h>
+
+#include "common.h"
+#include "unity_fixture.h"
+
+char data[DATA_SIZE];
```

Comment:
Is it necessary for this to be declared globally?

Reply:
It will be used much more in all future test cases and it is initialized by test setup.